sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -109,57 +109,9 @@ This starter provides following properties to be customized:
 | **azure.activedirectory**.user-group.allowed-groups              | Expected user groups that an authority will be granted to if found in the response from the MemeberOf Graph API Call. |
 
 ### Web application
-Based on Azure AD as a Web application, it uses OAuth2 authorization code flow to authentication, and authorizes resources based on the groups or roles claim in the access token.
+Based on Azure AD as a Web application, it uses OAuth2 authorization code flow to authentication, and authorizes resources based on the groups in the access token.
```

Comment:
We have 4 scenarios:
```
### Web application
### Web application with function of visiting resource server
### Resource server
### Resource server with function of on-behalf-of flow
```
Each scenario have these contents:
```
#### System diagram
#### Include the package
#### Add configurations application.yml
#### Write related Java code
```

**on-demand** is not a standalone scenario, delete it in the readme. Jus like many other configuration items: there is no detailed description in readme.